packages/manager/apps/cloud-connect/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@ovh-ux/manager-cloud-connect-app",
+  "version": "0.0.0",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/apps/cloud-connect"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "scripts": {
+    "build": "yarn build:eu && yarn build:ca && yarn build:us",
+    "build:ca": "webpack --env.production --env.region='CA' && mv dist dist-CA",
```

Comment:
Is this still needed ? Maybe we could use the `bootstrapApplication` provided by `@ovh-ux/manager-core` instead